scrapers/nytData.js

```diff
@@ -0,0 +1,36 @@
+const axios = require('axios');
+const csv = require('csvtojson');
+const logger = require('../utils/logger');
+const { updateCache } = require('../utils/nyt_cache');
+
+const US_COUNTY_DATA_URL = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv';
+const US_STATE_DATA_URL = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv';
+const US_NATION_WIDE_DATA_URL = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us.csv';
+const REDIS_KEYS = ['nyt_counties', 'nyt_states', 'nyt_USA'];
```

Comment:
you are using different redis keys than you propose in the config

Reply:
I think the label I attached is confusing here because I'm referencing the key in the `config.keys.json` file that retrieves the Redis key itself 😆 . Should probably label it `REDIS_CONFIG_KEYS` or something like that, but open to suggestions.

Reply:
no you were literally using "nyt_counties" in this file and using "nyt:counties" in config.keys.json. but i see you already fixed it! nice! i would still propose to not have them hardcoded in your scraper code. 
please have a look at the other scraper code, the server.js file passes the redis keys when calling the scrape function

Reply:
ahhhh i see!